README.md

```diff
@@ -141,61 +202,18 @@ Some hints are in this github issue: https://github.com/square/exchange/issues
 G❯ I want you to help me increase the test coverage in src/java... use mvn test to run the unit tests to check it works.
 ```
 
+## FAQ
 
-#### Advanced LLM config
-
-goose works on top of LLMs (you bring your own LLM). If you need to customize goose, one way is via editing: `~/.config/goose/profiles.yaml`. 
-
-It will look by default something like: 
-
-```yaml
-default:
-  provider: openai
-  processor: gpt-4o
-  accelerator: gpt-4o-mini
-  moderator: truncate
-  toolkits:
-  - name: developer
-    requires: {}
-```
-
-*Note: This requires the environment variable `OPENAI_API_KEY` to be set to your OpenAI API key. goose uses at least 2 LLMs: one for acceleration for fast operating, and processing for writing code and executing commands.*
-
-You can tell it to use another provider for example for Anthropic: 
-
-```yaml
-default:
-  provider: anthropic
-  processor: claude-3-5-sonnet-20240620
-  accelerator: claude-3-5-sonnet-20240620
-...
-```
-
-*Note: This will then use the claude-sonnet model, you will need to set the `ANTHROPIC_API_KEY` environment variable to your anthropic API key.* 
-
-For Databricks hosted models: 
-
-```yaml
-default:
-  provider: databricks
-  processor: databricks-meta-llama-3-1-70b-instruct
-  accelerator: databricks-meta-llama-3-1-70b-instruct
-  moderator: passive
-  toolkits:
-  - name: developer
-    requires: {}
-```
-
-This requires `DATABRICKS_HOST` and `DATABRICKS_TOKEN` to be set accordingly
+**Q:** Why did I get error message of "The model `gpt-4o` does not exist or you do not have access to it.` when I talked goose?
 
-(goose can be extended to support any LLM or combination of LLMs).
+**A:** You can find out the LLM provider and models in the configuration file `~/.config/goose/sessions` here to check whether your LLM provider account has access to the models.  For example, after you have made a successful payment of $5 or more (usage tier 1), you'll be able to access the GPT-4, GPT-4 Turbo, GPT-4o models via the OpenAI API. [How can I access GPT-4, GPT-4 Turbo, GPT-4o, and GPT-4o mini?](https://help.openai.com/en/articles/7102672-how-can-i-access-gpt-4-gpt-4-turbo-gpt-4o-and-gpt-4o-mini).
```

Comment:
did you mean ~/.config/goose/profile.yaml?

Reply:
oh! yes, I used the wrong file